target-bin/bootstrap-fixup.in

```diff
@@ -16,13 +16,15 @@ elif [ $DISTRIB_NAME = "Debian" ]; then
   # grub-legacy conflicts grub-pc dependencies
   # No grub-legacy on Ubuntu, just on Debian
   # Work around bcron-run conflict due to cron being removed
-  apt-get purge -y grub-legacy bcron-run &> /dev/null
+  # Needed for KVM, but apparently errors out for LXC, so the true ignores the error
+  apt-get purge -y grub-legacy bcron-run &> /dev/null || true
 fi
 echo '127.0.1.1 gitian' >> /etc/hosts
 
 # If LXC
 if grep /lxc/gitian /proc/1/cgroup > /dev/null; then
-  adduser --disabled-password --gecos ubuntu --quiet ubuntu || true
+  adduser --disabled-password --gecos ${DISTRIB_NAME,,} --quiet ${DISTRIB_NAME,,} || true
+  adduser ${DISTRIB_NAME,,} sudo
```

Comment:
Related: As I was setting up gitian-builder and observed that sudo was seemingly required to make the LXC image etc. I started looking at unprivileged LXC containers; there seem to be ways to set up the LXC environment to not need sudo:
* https://myles.sh/configuring-lxc-unprivileged-containers-in-debian-jessie/
* https://www.stgraber.org/2014/01/17/lxc-1-0-unprivileged-containers/